README.md

```
@@ -1,387 +1,207 @@
-<p align="left">
-    <strong>Example Live Demo: </strong> <a href="https://dev.mhmiton.com/laravel-modules-livewire-example" target="_blank">https://dev.mhmiton.com/laravel-modules-livewire-example</a>
-</p>
-
-### Installation:
-
-Install through composer:
+### Examples:
+<strong>Live Demo: </strong> <a href="https://dev.mhmiton.com/laravel-modules-livewire-example" target="_blank">https://dev.mhmiton.com/laravel-modules-livewire-example</a> - [Source Code](https://github.com/mhmiton/laravel-modules-livewire-example)
 
+## Installation:
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)